lib/linguist/languages.yml

```
@@ -550,6 +550,14 @@ Bicep:
   tm_scope: source.bicep
   ace_mode: text
   language_id: 321200902
+Bikeshed:
+  type: programming
```

This should be `markup`, not `programming`.

```suggestion
  type: markup
```